### PR TITLE
Use AutorestSDK Authorization package.

### PR DESCRIPTION
**Pull request by Monicalyy**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed. You can find a more complete discussion of PowerShell cmdlet best practices [here](https://msdn.microsoft.com/en-us/library/dd878270(v=vs.85).aspx).

- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md).**
- [ ] **If changes were made to any cmdlet, the XML help was regenerated using the [platyPSHelp module](https://github.com/Azure/azure-powershell/blob/dev/documentation/platyPSHelp-documentation.md).**
- [ ] **If any large changes are made to a service, they are reflected in the respective [change log](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#updating-the-change-log).**

### [General Guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] The pull request does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/dev/documentation/breaking-changes.md) (unless a major version change occurs in the assembly and module).

### [Testing Guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.
- [ ] PowerShell scripts used in tests should do any necessary setup as part of the test or suite setup, and should not use hard-coded values for locations or existing resources.

### [Cmdlet Signature Guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#cmdlet-signature-guidelines)
- [ ] New cmdlets that make changes or have side effects should implement `ShouldProcess` and have `SupportShouldProcess=true` specified in the cmdlet attribute. You can find more information on `ShouldProcess` [here](https://gist.github.com/markcowl/338e16fe5c8bbf195aff9f8af0db585d#what-is-the-change).
- [ ] Cmdlet specifies `OutputType` attribute if any output is produced - if the cmdlet produces no output, it should implement a `PassThru` parameter.

### [Cmdlet Parameter Guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#cmdlet-parameter-guidelines)
- [ ] Parameter types should not expose types from the management library - complex parameter types should be defined in the module.
- [ ] Complex parameter types are discouraged - a parameter type should be simple types as often as possible. If complex types are used, they should be shallow and easily creatable from a constructor or another cmdlet.
- [ ] Cmdlet parameter sets should be mutually exclusive - each parameter set must have at least one mandatory parameter not in other parameter sets.

### CODE REVIEW

**Comment by markcowl** [comment]
@Monicalyy You will need to resolve the CLA in order for us to look at this PR.  Either sign the CLA, or join the azure org, as appropriate.

File: src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Commands.ScenarioTests.ResourceManager.Common.csproj [hunk 1]
```diff
@@ -57,9 +57,8 @@
       <HintPath>..\..\..\packages\Microsoft.Azure.Management.Resources.2.20.0-preview\lib\net40\Microsoft.Azure.ResourceManager.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="Microsoft.Azure.Test.Framework">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\..\packages\Microsoft.Azure.Test.Framework.1.0.6179.26854-prerelease\lib\net45\Microsoft.Azure.Test.Framework.dll</HintPath>
```
Comment (darshanhs90) [inline src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Commands.ScenarioTests.ResourceManager.Common.csproj]: >ork.1.0.6179.26854-pr [](start = 61, length = 21)

donot modify this to point to earlier framework

File: src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Commands.ScenarioTests.ResourceManager.Common.csproj [hunk 2]
```diff
@@ -73,8 +72,7 @@
       <HintPath>..\..\..\packages\Microsoft.IdentityModel.Clients.ActiveDirectory.2.28.3\lib\net45\Microsoft.IdentityModel.Clients.ActiveDirectory.WindowsForms.dll</HintPath>
     </Reference>
     <Reference Include="Microsoft.Rest.ClientRuntime, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.Rest.ClientRuntime.2.3.5\lib\net45\Microsoft.Rest.ClientRuntime.dll</HintPath>
-      <Private>True</Private>
+      <HintPath>..\..\..\packages\Microsoft.Rest.ClientRuntime.2.3.6\lib\net45\Microsoft.Rest.ClientRuntime.dll</HintPath>
```
Comment (darshanhs90) [inline src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Commands.ScenarioTests.ResourceManager.Common.csproj]: >.3.6\lib\net4 [](start = 64, length = 13)

why was this modified?
any reason behind this?

File: src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Commands.ScenarioTests.ResourceManager.Common.csproj [hunk 3]
```diff
@@ -170,6 +168,7 @@
     <Compile Include="XunitTracingInterceptor.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
```
Comment (darshanhs90) [inline src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Commands.ScenarioTests.ResourceManager.Common.csproj]: > <None Include="app.config" /> [](start = 3, length = 30)

included why?

File: src/ResourceManager/Resources/Commands.Resources.Test/Commands.Resources.Test.csproj [hunk 4]
```diff
@@ -78,8 +78,8 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\packages\Microsoft.Azure.Management.Resources.2.20.0-preview\lib\net40\Microsoft.Azure.ResourceManager.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.Azure.Test.Framework">
-      <HintPath>..\..\..\packages\Microsoft.Azure.Test.Framework.1.0.6179.26854-prerelease\lib\net45\Microsoft.Azure.Test.Framework.dll</HintPath>
+    <Reference Include="Microsoft.Azure.Test.Framework, Version=1.0.6052.28118, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.Azure.Test.Framework.1.0.6052.28118-prerelease\lib\net45\Microsoft.Azure.Test.Framework.dll</HintPath>
```
Comment (darshanhs90) [inline src/ResourceManager/Resources/Commands.Resources.Test/Commands.Resources.Test.csproj]: donot modify this to point to earlier framework

File: src/ResourceManager/Resources/Commands.Resources.Test/Commands.Resources.Test.csproj [hunk 5]
```diff
@@ -115,9 +114,8 @@
       <HintPath>..\..\..\packages\Microsoft.Rest.ClientRuntime.Azure.Authentication.2.2.9-preview\lib\net45\Microsoft.Rest.ClientRuntime.Azure.Authentication.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="Microsoft.Rest.ClientRuntime.Azure.TestFramework">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\..\packages\Microsoft.Rest.ClientRuntime.Azure.TestFramework.1.5.0-preview\lib\net45\Microsoft.Rest.ClientRuntime.Azure.TestFramework.dll</HintPath>
+    <Reference Include="Microsoft.Rest.ClientRuntime.Azure.TestFramework, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.Rest.ClientRuntime.Azure.TestFramework.1.3.5-preview\lib\net45\Microsoft.Rest.ClientRuntime.Azure.TestFramework.dll</HintPath>
```
Comment (darshanhs90) [inline src/ResourceManager/Resources/Commands.Resources.Test/Commands.Resources.Test.csproj]: donot modify this to point to earlier framework

File: src/ResourceManager/Resources/Commands.Resources.Test/Commands.Resources.Test.csproj [hunk 6]
```diff
@@ -267,6 +266,7 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
```
Comment (darshanhs90) [inline src/ResourceManager/Resources/Commands.Resources.Test/Commands.Resources.Test.csproj]: ><None Include="app.config" /> [](start = 4, length = 29)

similar to the other comment

File: src/ResourceManager/Resources/Commands.Resources.Test/packages.config [hunk 7]
```diff
@@ -7,11 +7,11 @@
   <package id="Microsoft.Azure.Graph.RBAC" version="3.4.0-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.Insights" version="0.10.0-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Authorization" version="1.1.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.Authorization" version="2.5.0-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.ResourceManager" version="1.6.0-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.Resources" version="2.20.0-preview" targetFramework="net45" />
-  <package id="Microsoft.Azure.Test.Framework" version="1.0.6179.26854-prerelease" targetFramework="net45" />
-  <package id="Microsoft.Azure.Test.HttpRecorder" version="1.6.7-preview"  targetFramework="net45" />
```
Comment (darshanhs90) [inline src/ResourceManager/Resources/Commands.Resources.Test/packages.config]: donot modify this to point to earlier framework

File: src/ResourceManager/Resources/Commands.Resources.Test/packages.config [hunk 8]
```diff
@@ -20,10 +20,10 @@
   <package id="Microsoft.Data.Services.Client" version="5.6.4" targetFramework="net45" />
   <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.28.3" targetFramework="net45" />
   <package id="Microsoft.Net.Http" version="2.2.29" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime" version="2.3.5" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.6" targetFramework="net452" />
   <package id="Microsoft.Rest.ClientRuntime.Azure" version="3.3.5" targetFramework="net45" />
   <package id="Microsoft.Rest.ClientRuntime.Azure.Authentication" version="2.2.9-preview" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime.Azure.TestFramework" version="1.5.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime.Azure.TestFramework" version="1.3.5-preview" targetFramework="net452" />
```
Comment (darshanhs90) [inline src/ResourceManager/Resources/Commands.Resources.Test/packages.config]: donot modify this to point to earlier framework

File: src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/DeploymentTests.ps1 [hunk 9]
```diff
@@ -372,7 +372,7 @@ function Test-NewDeploymentWithInvalidParameters
 	# Setup
 	$rgname = Get-ResourceGroupName
 	$rname = Get-ResourceName
-	$rglocation = "CentralUSEUAP"
+	$rglocation = "centralus"
```
Comment (darshanhs90) [inline src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/DeploymentTests.ps1]: >$rglocation = "centralus" [](start = 1, length = 25)

can we make this a constant for this file,as it i used in all functions

File: src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/PolicyTests.ps1 [hunk 10]
```diff
@@ -35,7 +35,7 @@ function Test-PolicyDefinitionCRUD
 
 	New-AzureRMPolicyDefinition -Name test2 -Policy "{""if"":{""source"":""action"",""equals"":""blah""},""then"":{""effect"":""deny""}}"
 	$list = Get-AzureRMPolicyDefinition
-	Assert-AreEqual 2 @($list).Count
+	Assert-AreEqual 83 @($list).Count
```
Comment (darshanhs90) [inline src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/PolicyTests.ps1]: >Assert-AreEqual 83 @($list).Count [](start = 1, length = 33)

this changes everytime the create is run,be sure it runs in playback mode also,if not revert it back

**Comment by markcowl** [comment]
@Monicalyy Please resolve the CLA issue (either join the azure org or sign the CLA) and respond to Hari's feedback

**Comment by markcowl** [comment]
@azuresdkci add to whitelist

**Comment by cormacpayne** [comment]
Closing this pull request due to a lack of activity. @Monicalyy please feel free to reopen this PR when you are able to resolve the requested feedback.
